### PR TITLE
Removes BoH bomb confirmation dialog for antags

### PR DESCRIPTION
**Pull request by Cyberboss**
Minor QoL thing

:cl:
tweak: Antags will not receive a confirmation prompt when Bag of Holding bombing
/:cl:

### CODE REVIEW

**Comment by kevinz000** [comment]
Wtf I don't want to accidently bohbomb as antag

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -72,9 +72,10 @@
 
 /obj/item/storage/backpack/holding/handle_item_insertion(obj/item/W, prevent_warning = 0, mob/user)
 	if((istype(W, /obj/item/storage/backpack/holding) || count_by_type(W.GetAllContents(), /obj/item/storage/backpack/holding)))
-		var/safety = alert(user, "Doing this will have extremely dire consequences for the station and its crew. Be sure you know what you're doing.", "Put in [name]?", "Proceed", "Abort")
-		if(safety == "Abort" || !in_range(src, user) || !src || !W || user.incapacitated())
-			return
+		if(user && user.mind && user.mind.is_antag())
```
Comment (duncathan) [inline code/game/objects/items/storage/backpack.dm]: think very carefully about this check cyber

**Review by duncathan — DISMISSED** [review]
tbh idk if this is a good idea either way, but it's definitely bad as it is rn lmao

**Review by duncathan — DISMISSED** [review]
clicked the wrong fuckin button grr

**Comment by optimumtact** [comment]
Can we just remove it completely?

This is the one change that I literally hate because of how hugbox it is.

**Comment by Darkgenerallord** [comment]
I mostly use this so people can't use the bullshit 'I didn't know it would spawn a blackhole!!' excuse.

**Comment by optimumtact** [comment]
just call it what it is, bullshit

**Comment by XDTM** [comment]
To be fair if in some round bohs are mass-distributed some new player is going to try to be clever and stack inventories inside each other, it's not necessarily just griefing

**Comment by Bawhoppen** [comment]
@XDTM If the admin asks them, they can just say they didn't know. (and it will be funny)

**Comment by ShizCalev** [comment]
Changelog please. I know I myself used to hold that dialog open when I was planning to quickly run into somewhere to suicide BOH.

**Comment by Mey-Ha-Zah** [comment]
Do not remove that prompt, yes sometimes it can delay doing it, but that is favorable to mass grief because "I didn't know"

**Comment by duncathan** [comment]
I don't think this is worthwhile. Either everyone should get the prompt or nobody should, but I don't like it halfway. Personally I prefer keeping the prompt but I wouldn't mind it being completely gone either

**Pull request closed without merging**